docs/development/pigs/pig-016.rst

```diff
@@ -0,0 +1,215 @@
+estimation code is split into gammapy/background and gammapy/cube without a
+clear separation what goes where. So the choice was either to move everything to
+dissolve gammapy.background or to move all background-related code to
+gammapy.background.
+
+Looking at the package structure illustration above, it's clear that
+gammapy.background is problematic, because it is lower-level than
+gammapy.spectrum and gammapy.maps, so moving background estimation classes from
+gammapy.cube down to gammapy.background would likely lead to circular
+dependencies, because they rely on the map dataset class in gammapy.cube.
+
+The reflected region background code is not a great fit in gammapy/spectrum or
+anywhere in Gammapy except for gammapy.background, but moving it there does make
+sense if we say that all 1D spectrum code goes in gammapy.spectrum and all
+map-related analysis code goes in gammapy.cube.
+
+Dissolve gammapy.image
+----------------------
+
+The gammapy.image package was introduced early and contained a SkyImage class
+for 2D sky images, and 2D image models, and some image analysis helper functions
+such as code to make a radial profile from an image. Much later gammapy.maps was
+introduced and we managed to support 2D and 3D maps with one container instead
+of separate gammapy.image.SkyImage and gammapy.cube.SkyCube.
+
+Now, if we move gammapy.image.models to gammap.modeling as proposed above, and
+that SkyImage has been removed in 2018, there is very little functionality left
+in gammapy.image, and it's not clear what it's scope is.
+
+So we propose to clean this up, roughly like this:
+
+- asmooth.py and utils.py -> move to gammapy.cube
+- measure.py and profile.py -> replace with docs example or move to maps or cube
+- plotting.py -> replace with docs example or move to maps
+
+Clean up ``gammapy.utils``
+--------------------------
+
+Above we propose to move gammapy.utils.fitting, the biggest piece of
+gammapy.utils to gammapy.modeling. We still need and want to keep gammapy.utils,
+as a lowest-level layer in Gammapy where we write adapters and helpers for
+functionality e.g. from Numpy and Astropy. Mostly this is for usage within
+Gammapy, users very rarely need to import from gammapy.utils.
+
+Checking via ``ack gammapy.utils tutorials | egrep -v fitting`` we see that
+currently only energy_logspace, sqrt_space, sample_powerlaw and
+SphericalCircleSkyRegion are used from end-user documentation, i.e. very little.
+
+We propose to clean up gammapy.utils, e.g. gammapy/utils/coordinates.py still
+contains coordinate helper functions that were added before astropy.coordinates
+existed. We do not detail the cleanup here, this will be discussed on a
+case-by-case basis. After that we will review what is left and decide whether we
+keep or completely remove gammapy.utils from the Gammapy docs, i.e. if it's
+public API or not.
+
+Outlook
+=======
+
+During discussion of this PIG, several other ideas surfaced and were considered.
```

Comment:
This paragraph is very chatty and I think statements like "either we will do it or not" are not really helpful inf focusing the development work. So  maybe shorten the paragraph to 2 sentences with a more concrete proposal? E.g.:

During discussion of this PIG, several other ideas surfaced. The following ideas should be considered and evaluated for feasibility (as done in `GH 2290`_)  in future:

Reply:
done